app/src/main/java/com/kickstarter/ui/activities/ProjectPageActivity.kt

```diff
@@ -1085,9 +1123,96 @@ class ProjectPageActivity :
         }
     }
 
+    // Update the UI with the returned PaymentOption
```

Comment:
For everything related with the PaymentSheet, we should try instead of copy and paste the code from PledgeFragment, to create some kind of Extension function or utils for Stripe's integration in a way the is re-usable.